lib/mauth/client/signer.rb

```diff
@@ -31,24 +33,30 @@ def signed_headers(object, attributes = {})
       def signed_headers_v1(object, attributes = {})
         attributes = { time: Time.now.to_i.to_s, app_uuid: client_app_uuid }.merge(attributes)
         string_to_sign = object.string_to_sign_v1(attributes)
```

Comment:
Not new in this PR, but it seems that `hashed_string_to_sign` is more descriptive than `string_to_sign` -- what do you think?